compose.yaml

```diff
@@ -37,7 +37,7 @@ services:
       - etl-gpu
 
   ctakes-covid-base:
-    image: smartonfhir/ctakes-covid:1.1.0
+    image: smartonfhir/ctakes-covid:1.1.1
```

Comment:
This is very tight pinning, I know. And maybe it should just be `1.1`? But I also kind of like how paranoid it is. Open to thoughts.

Reply:
i'm ok in this case since its unlikely to change, and we always want to run latest.

Reply:
You're OK with what? `1.1` or the paranoia?

While we always want to run the latest, if there's every any change that can affect the results from cTAKES, we need to bump some task version numbers in every task that uses cTAKES.

So like, I wouldn't want to use just `1` or `latest` as the tag. And I went extra paranoid by also wanting to consider every patch release for possible changes. Which is maybe too much, since we are co-maintainers of that docker image and can enforce semantic versioning. But only co-maintainers...

Reply:
(we need to bump task version numbers to (a) reset any existing ctakes caches you may have and (b) flag new task results so you can track which bits need to be re-run with the new NLP changes)